.eslintrc.js

```diff
@@ -9,8 +9,9 @@ module.exports = {
     "eslint:recommended",
     "plugin:ava/recommended",
     "plugin:@typescript-eslint/recommended",
-    "plugin:prettier/recommended",
-    "prettier/@typescript-eslint",
```

Comment:
These are a pain to keep up to date. Recommendation is to not use them.

Seems to work fine in vscode, and added a check to the lint script

(also we weren't even running lint on CI before!)